zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java

```diff
@@ -524,4 +528,67 @@ public String generateForm(HttpMessage msg) throws UnsupportedEncodingException
         HistoryReference createHistoryReference(int id)
                 throws DatabaseException, HttpMalformedHeaderException;
     }
+
+    public void injectCsrfToken(HttpMessage message, HttpRequestConfig config) {
```

Comment:
- I'd suggest using the "old" name`regenerateAntiCsrfToken`, it's not injecting (just replacing and clear with "anti").
 - The two methods can be merged (and feel free to clean up).

Reply:
Also worth adding some JavaDoc (and since tag).